lib/workers/repository/update/pr/changelog/source-github.ts

```diff
@@ -195,7 +195,7 @@ function findTagOfRelease(
   tags: string[]
 ): string | undefined {
   const regex = regEx(`(?:${packageName}|release)[@-]`, undefined, false);
-  const excactReleaseRegex = regEx(`${packageName}[@-_]v?${depNewVersion}`);
+  const excactReleaseRegex = regEx(`${packageName}[@\\-_]v?${depNewVersion}`);
```

Comment:
```suggestion
  const excactReleaseRegex = regEx(`${packageName}[@_-]v?${depNewVersion}`);
```
this would be a simpler solution 😬